notebooks/240-efficientdet-object-detection/240-efficientdet-object-detection.ipynb

```diff
@@ -0,0 +1,666 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #27.            output_keys = [model.output(4),  # CLASS KEY
</pre><p>They were already assigned. Why does it happen again?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>